test/integration/query-interface/schemas.test.ts

```diff
@@ -0,0 +1,20 @@
+import { expect } from 'chai';
+import { sequelize, getTestDialect } from '../support';
+
+const queryInterface = sequelize.queryInterface;
+
+const testSchema = 'testSchema';
+const dialect = getTestDialect();
+
+if (dialect.startsWith('postgres')) {
+  describe('QueryInterface#{create,drop}Schema', async () => {
+    it('should create then drop a schema', async () => {
+      const preCreationSchemas: string[] = await queryInterface.showAllSchemas();
+      await queryInterface.createSchema(testSchema);
+      expect(preCreationSchemas).to.not.include(testSchema);
```

Comment:
this expect should run before createSchema, as well as this change:

```suggestion
      expect(preCreationSchemas).to.not.include(testSchema, 'testSchema already exists prior to creation');
```